openapi-docs/docs/DefaultApi.md

```diff
@@ -0,0 +1,224 @@
+ **cvc** | **String**| Cvc number behind credit card | 
+ **expMonth** | **String**| 2 digit expected credit card expiry month.  | 
+ **expYear** | **String**| 4 digit expected credit card expiry year.  | 
+ **email** | **String**| Subscriber&#39;s email address | 
+ **accessControlAllowOrigin** | **String**|  | 
+ **accessControlAllowHeaders** | **String**|  | 
+ **contentType** | **String**|  | 
+
+### Return type
+
+null (empty response body)
+
+### Authorization
+
+No authorization required
+
+### HTTP request headers
+
+- **Content-Type**: application/x-www-form-urlencoded
+- **Accept**: Not defined
+
+
+## subscription
+
+> subscription(subscriptionId, accessControlAllowOrigin, accessControlAllowHeaders, contentType)
+
+subscription
+
+Delete a user&#39;s subscription on Stripe, either active or expired.
+
+### Example
+
+```javascript
+import AmbianicFunctionsCollection from 'ambianic_functions_collection';
+
+let apiInstance = new AmbianicFunctionsCollection.DefaultApi();
+let subscriptionId = SUBSCRIBER-ID; // String | ID for the user's subscription to be deleted
+let accessControlAllowOrigin = *; // String | 
+let accessControlAllowHeaders = Content-Type; // String | 
+let contentType = application/json; // String | 
+apiInstance.subscription(subscriptionId, accessControlAllowOrigin, accessControlAllowHeaders, contentType, (error, data, response) => {
+  if (error) {
+    console.error(error);
+  } else {
+    console.log('API called successfully.');
+  }
+});
+```
+
+### Parameters
+
+
+Name | Type | Description  | Notes
+------------- | ------------- | ------------- | -------------
+ **subscriptionId** | **String**| ID for the user&#39;s subscription to be deleted | 
+ **accessControlAllowOrigin** | **String**|  | 
+ **accessControlAllowHeaders** | **String**|  | 
+ **contentType** | **String**|  | 
+
+### Return type
+
+null (empty response body)
+
+### Authorization
+
+No authorization required
+
+### HTTP request headers
+
+- **Content-Type**: Not defined
+- **Accept**: Not defined
+
+
+## subscriptionData
+
+> subscriptionData(userStripeId, userSubscriptionId, accessControlAllowOrigin, accessControlAllowHeaders, contentType)
+
+subscription-data
+
+Retrieve a subscribed customer&#39;s data from Stripe
+
+### Example
+
+```javascript
+import AmbianicFunctionsCollection from 'ambianic_functions_collection';
+
+let apiInstance = new AmbianicFunctionsCollection.DefaultApi();
+let userStripeId = STRIPE_CUSTOMER_ID; // String | Id for the customer created on Stripe
+let userSubscriptionId = CREATED_SUBSCRIPTION_ID; // String | Id for the user's subscription with Ambianic
+let accessControlAllowOrigin = *; // String | 
+let accessControlAllowHeaders = Content-Type; // String | 
+let contentType = application/json; // String | 
+apiInstance.subscriptionData(userStripeId, userSubscriptionId, accessControlAllowOrigin, accessControlAllowHeaders, contentType, (error, data, response) => {
```

Comment:
@vickywane a better name for this JS function would be
`apiInstance.getSubscriptionData()`

Reply:
The `operationId` has been adjusted to implement this.